Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -313,7 +331,9 @@ private extension ARGeoTrackingConfiguration {
         }
     }
 }
+#endif
 
+@available(visionOS, unavailable)
 private extension WorldScaleSceneView {
```

Comment:
`WorldScaleSceneView` is annotated as unavailable on visionOS, so why is the conditional compilation necessary.